src/overlays/actors/ovl_Bg_Bdan_Objects/z_bg_bdan_objects.c

```diff
@@ -278,63 +297,64 @@ void func_8086C3D8(BgBdanObjects* this, PlayState* play) {
     }
 }
 
-void func_8086C55C(BgBdanObjects* this, PlayState* play) {
+void BgBdanObjects_OctoPlatform_PauseBeforeDescending(BgBdanObjects* this, PlayState* play) {
     this->timer--;
 
     if (this->timer == 0) {
         Flags_UnsetSwitch(play, this->switchFlag);
     } else if (this->timer == -40) {
         this->timer = 0;
-        this->actionFunc = func_8086C3D8;
+        this->actionFunc = BgBdanObjects_OctoPlatform_DescendWithBigOcto;
     }
 }
 
-void func_8086C5BC(BgBdanObjects* this, PlayState* play) {
+void BgBdanObjects_OctoPlatform_WaitForBigOctoToStartBattle(BgBdanObjects* this, PlayState* play) {
     if (this->timer != 0) {
         this->timer--;
     }
     if ((this->timer == 0) && (this->dyna.actor.child != NULL)) {
         if (this->dyna.actor.child->params == 2) {
-            this->actionFunc = func_8086C618;
+            this->actionFunc = BgBdanObjects_OctoPlatform_BattleInProgress;
         } else if (this->dyna.actor.child->params == 0) {
             this->dyna.actor.child->params = 1;
         }
     }
 }
 
-void func_8086C618(BgBdanObjects* this, PlayState* play) {
+void BgBdanObjects_OctoPlatform_BattleInProgress(BgBdanObjects* this, PlayState* play) {
     Collider_UpdateCylinder(&this->dyna.actor, &this->collider);
     CollisionCheck_SetAT(play, &play->colChkCtx, &this->collider.base);
     if (Flags_GetClear(play, this->dyna.actor.room)) {
         Flags_SetSwitch(play, this->switchFlag);
         this->dyna.actor.home.rot.y = (s16)(this->dyna.actor.shape.rot.y + 0x2000) & 0xC000;
-        this->actionFunc = func_8086C6EC;
+        this->actionFunc = BgBdanObjects_SinkToFloorHeight;
     } else {
         this->dyna.actor.shape.rot.y += this->dyna.actor.world.rot.y;
+        // This plays a sound effect id (0x2063) that is not present in environmentbank_table?
         func_800F436C(&this->dyna.actor.projectedPos, 0x2063, ABS(this->dyna.actor.world.rot.y) / 512.0f);
```

Comment:
@engineer124 if you're interested (if not interesting/interested feel free to ignore)

(is this just `NA_SE_EV_ROLL_STAND - SFX_FLAG` ?)

Reply:
I think you're right about this, considering 
1) `NA_SE_EV_ROLL_STAND` is otherwise unused
2) related sound effect `NA_SE_EV_ROLL_STAND_2` is always used with SFX_FLAG

going to change it in upcoming commit